taskvine/src/worker/vine_worker.c

```diff
@@ -552,62 +640,21 @@ static void reap_process(struct vine_process *p, struct link *manager)
 
 	itable_remove(procs_running, p->task->task_id);
 	itable_insert(procs_complete, p->task->task_id, p);
-}
-
-/*
-Transmit the results of the given process to the manager.
-*/
-
-static void report_task_complete(struct link *manager, struct vine_process *p)
-{
-	int64_t output_length;
 
 	int output_file = open(p->output_file_name, O_RDONLY);
 	if (output_file >= 0) {
 		struct stat info;
 		fstat(output_file, &info);
-		output_length = info.st_size;
-	} else {
-		output_length = 0;
-	}
-
-	send_message(manager,
-			"result %d %d %lld %llu %llu %d\n",
-			p->result,
-			p->exit_code,
-			(long long)output_length,
-			(unsigned long long)p->execution_start,
-			(unsigned long long)p->execution_end,
-			p->task->task_id);
-
-	if (output_file >= 0) {
-		link_stream_from_fd(manager, output_file, output_length, time(0) + options->active_timeout);
+		p->output_length = info.st_size;
 		close(output_file);
+	} else {
+		p->output_length = 0;
 	}
 
 	total_task_execution_time += (p->execution_end - p->execution_start);
 	total_tasks_executed++;
 }
 
-/*
-For every unreported complete task and watched file,
-send the results to the manager.
-*/
-
-static void report_tasks_complete(struct link *manager)
-{
-	struct vine_process *p;
-
-	while ((p = itable_pop(procs_complete))) {
-		report_task_complete(manager, p);
-	}
-
-	vine_watcher_send_changes(watcher, manager, time(0) + options->active_timeout);
```

Comment:
Hmm, I think this was lost without replacement.  The manager still needs a way to request updates re watched files.  Is `vine_watcher_send_changes` still called from somewhere?

Reply:
re-added updates for watched files.